src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -269,18 +297,6 @@ def check_uid_uniqueness(proof_dir, proof_uids):
     sys.exit(1)
 
 
-def should_enable_memory_profiling(litani_caps, args):
```

Comment:
Why remove this?

Reply:
Resolving since reasoning here is the same as with other changes related to no longer checking for Litani capabilities. I'm removing this change.